### PR TITLE
Jordan/1603 sidebar links

### LINKED ISSUE #1603: staking and governance do not appear selected in the sidebar
1. click on staking / governance links in sidebar
2. see arrow is not pink, text is not white, background is not lightened

### PR DESCRIPTION
**Pull request by jbibla**
Closes #1603 

_Description:_

- fixed sidebar bug
- fixed some other errors
- made some parts of the app more consistent / minor style UPGRADEZZ

❤️ Thank you!

---

* [x] Added entries in `CHANGELOG.md` with issue # and GitHub username
* [x] Reviewed `Files changed` in the github PR explorer

### CODE REVIEW

File: app/src/renderer/components/staking/TabMyDelegations.vue [hunk 1]
```diff
@@ -16,7 +12,7 @@
       |
       router-link(:to="{name: 'Validators'}") the validator list
       |
-      | to spread some of your Atoms around.
```
Comment (fedekunze) [inline app/src/renderer/components/staking/TabMyDelegations.vue]: lol why did we had `spread`

File: app/src/renderer/components/governance/LiProposal.vue [hunk 2]
```diff
@@ -26,8 +31,7 @@ export default {
       if (this.proposal.proposal_status === `Passed`)
         return {
           button: null,
-          message: `This proposal has passed`,
-          color: `green`
```
Comment (fedekunze) [inline app/src/renderer/components/governance/LiProposal.vue]: I think we should leave it green here

File: app/src/renderer/components/governance/LiProposal.vue [hunk 3]
```diff
@@ -45,7 +49,7 @@ export default {
         return {
           button: `vote`,
           message: `Voting for this proposal is open`,
-          color: `blue`
+          color: `green`
```
Comment (fedekunze) [inline app/src/renderer/components/governance/LiProposal.vue]: and use orange or similar color here

File: app/src/renderer/components/governance/LiProposal.vue (hunk 2 above)
Comment (jbibla) [reply app/src/renderer/components/governance/LiProposal.vue]: it's not a little green dot anymore - it's a green checkmark.

if felt the green status dot should be for signalling "voting"

File: app/src/renderer/components/governance/LiProposal.vue (hunk 3 above)
Comment (jbibla) [reply app/src/renderer/components/governance/LiProposal.vue]: no green is good for open voting! like a traffic light! GO GO GO!

**Comment by jbibla** [comment]
> PageWallet's style looks more consistent now with the rest of the app. I still have a feeling that it's missing something...

totally agree. trying to stay focussed on our milestones 😄

**Pull request merged**